client/components/Copyright.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+
+import { Link } from 'react-router-dom';
+
+export function Copyright() {
+  return (
+    <Typography variant='body2' color='textSecondary' align='center'>
+      {'Copyright © '}
+      <Link color='inherit' to='/'>
+        PopkWiz
```

Comment:
Nit: typo

Reply:
`PopkWiz` should be `PopKwiz`. Typo fixed.